libs/features/seotech/seotech.js

```diff
@@ -0,0 +1,34 @@
+export const SEOTECH_API_URL_PROD = 'https://14257-seotech.adobeioruntime.net';
+export const SEOTECH_API_URL_STAGE = 'https://14257-seotech-stage.adobeioruntime.net';
+
+export function logError(msg) {
+  window.lana.log(`SEOTECH: ${msg}`, {
+    debug: false,
+    implicitSampleRate: 100,
+    sampleRate: 100,
```

Comment:
Is this used on pages that don't see a lot of traffic? A sample rate of 100 sounds too high - usually 1 is enough

Reply:
This feature won't be used at scale until late Q4/Q1. We want all the errors logged so I can get some usefulness from reporting. This also makes it easier to dev an alert, e.g. "/this/page can't find /this/video" x 10. I can always come back later and lower these.

Slightly OT, but I've checked LANA and I'm concerned Milo is sampling far too little by default.

Reply:
Yeah, as long as you have your use-case in mind that's good 👍 

As for the OT: Milo currently gets a lot of traction and a lot of projects that go live / will go live very soon. On the AEM lana logs things got flooded really fast, so it's just a matter of time until milo will see equal amounts of logs